metacat-common-server/src/main/java/com/netflix/metacat/common/server/spi/MetacatCatalogConfig.java

```diff
@@ -104,6 +89,17 @@ public boolean isThriftInterfaceRequested() {
      * Properties in the catalog.
      */
     public static class Keys {
+        /**
+         * Catalog name. Usually catalog name is derived from the name of the catalog properties file.
+         * One could also specify it in the properties under the property name <code>catalog.name</code>.
+         * For example, if a catalog has two shards with catalog defined in two property files,
+         * then we can have the <code>catalog.name</code> set to one name.
+         */
+        public static final String CATALOG_NAME = "catalog.name";
+        /**
+         * Connector type.
```

Comment:
is this type or name?

Reply:
This is the connector type. We had this property as connector.name. Probably we had this inherited from presto.